definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "LiveFrom": "24/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServiceApplicationDocuments",
+    "Label": "Service Application Documents",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "DivorceServiceApplicationDocument",
+    "SecurityClassification": "Public"
+  },
```

Comment:
New document collection

Reply:
I think this should be just a collection of complex type <DivorceDocument>. 

Pretty much the same as:

```
  {
    "LiveFrom": "01/01/2017",
    "CaseTypeID": "DIVORCE",
    "ID": "D8DocumentsGenerated",
    "Label": "Documents generated",
    "FieldType": "Collection",
    "FieldTypeParameter": "DivorceDocument",
    "SecurityClassification": "Public"
  },
```